Automattic/wp-calypso
# Marketplace Reviews: Add the avatar to the Reviews List

Fixes #85431

## Proposed Changes

Add the avatar to the Reviews List

## Testing Instructions

* On this branch 
* Go to a plugin details page with reviews. Ex: `/plugins/woocommerce-bookings/`
* On the bottom, click on `Read all reviews`
* Check if the avatar are being displayed

![CleanShot 2023-12-19 at 14 14 16@2x](https://github.com/Automattic/wp-calypso/assets/5039531/fc19ab62-edb3-4a7f-b926-22bb731c1676)


## Pre-merge Checklist
